### PR TITLE
Create template

### PR DESCRIPTION
**Pull request by Modulos**

### CODE REVIEW

File: opennebula_api/models.py [hunk 1]
```diff
@@ -60,15 +61,50 @@ def _get_opennebula_client(self, username, password):
...
             return opennebula_user
         except WrongNameError as wrong_name_err:
             opennebula_user = self.oneadmin_client.call(oca.User.METHODS['allocate'], email,
                                                         password, 'core')
+        except ConnectionRefusedError:
+            print('Could not connect to host: {host} via protocol {protocol}'.format(
+                    host=settings.OPENNEBULA_DOMAIN,
+                    protocol=settings.OPENNEBULA_PROTOCOL)
+                )
             return opennebula_user
```
Comment (pcoder) [inline opennebula_api/models.py]: Why are we returning opennebula_user here?

Comment (pcoder) [inline opennebula_api/models.py]: I think we should have a better way of propagating the "ConnectionRefusedError" to the view?

Comment (Modulos) [inline opennebula_api/models.py]: This is a typo, thanks :+1:

Comment (Modulos) [inline opennebula_api/models.py]: Yes your right. Can you add a logger and propagate exceptions?

**Comment by levivm** [comment]
@Modulos it has a conflict in `models.py` can u fix it ? plz

**Pull request merged**